client/app/scripts/components/details-card.js

```diff
@@ -0,0 +1,53 @@
+import React from 'react';
+
+import NodeDetails from './node-details';
+
+// card dimensions in px
+const marginTop = 24;
+const marginBottom = 48;
+const marginRight = 36;
+const panelWidth = 420;
+const offset = 8;
+
+export default class DetailsCard extends React.Component {
+
+  constructor(props, context) {
+    super(props, context);
+    this.state = {
+      mounted: null
+    };
+  }
+
+  componentDidMount() {
+    setTimeout(() => {
+      this.setState({mounted: true});
+    });
```

Comment:
While this pattern is very clear and simple it feels a tiny bit hackish, does `ReactCSSTransitionGroup` etc give you a nice way to specify the animation states? Skimming the docs it look like it might, but might complicate things a lot too.